android/app/build.gradle

```diff
@@ -393,6 +393,8 @@ dependencies {
   implementation 'com.google.guava:guava:33.1.0-android'
   implementation 'com.github.devnullorthrow:MPAndroidChart:3.2.0-alpha'
   implementation 'net.jcip:jcip-annotations:1.0'
+  // Used for circular profile image in profile
+  implementation 'de.hdodenhof:circleimageview:3.1.0'
```

Comment:
Would a square image be enough? It would be great to avoid non-critical dependencies.